src/Pages/Home.tsx

```diff
@@ -0,0 +1,3 @@
+export const Home = () => (
```

Comment:
```suggestion
export const HomePage = () => (
```

The same with file name